schema/appdata/mux.go

```
@@ -139,7 +139,7 @@ func ListenerMux(listeners ...Listener) Listener {
 
 	mux.onBatch = func(batch PacketBatch) error {
 		for _, listener := range listeners {
-			err := batch.apply(&listener) //nolint:gosec // aliasing is safe here
+			err := batch.apply(&listener)
```

golangci-lint (1.62.2)

[medium] G601: Implicit memory aliasing in for loop.

(gosec)